db/db_iterator_test.cc

```diff
@@ -3824,6 +3824,117 @@ TEST_F(DBIteratorTest, IteratorsConsistentViewExplicitSnapshot) {
   }
 }
 
+TEST_P(DBIteratorTest, MemtableTombstoneScanLimitWithSeek) {
+  // Tests that option memtable_tombstone_scan_limit works when the limit
+  // is reached during a Seek() operation.
+  const int kNumDelete = 10;
+  for (int limit : {kNumDelete, kNumDelete + 1}) {
+    Options options;
+    options.create_if_missing = true;
+    options.memtable_tombstone_scan_limit = limit;
+    DestroyAndReopen(options);
+    for (int i = 0; i < kNumDelete; ++i) {
```

Comment:
Please do a sequence of Put(s) and then a corresponding sequence of Delete(s) to make the test more realistic.

Reply:
Updated.